### DISCUSSION TITLE
Splitting the workload

### DISCUSSION THREAD

**Discussion by StutzTAFE** [Felix-E-S/T24S2T1EXTG08#2] [General]
Hey Felix,

I'm happy to work on the currency conversion calculator if you're happy to do the mortgage calculator

Thanks, 
Jethro

**Comment by Felix-E-S** [comment]
Hi Jethro,
Sounds good! Lets do it that way then. I will work on the mortgage calculator.
Let me know if you need help with anything or if I missed anything in the set up.
Cheers,
Felix
